frontend/scripts/react-components/profile-root/profile-root.scss

```diff
@@ -1,6 +1,73 @@
 @import 'styles/settings';
 
 .c-profile-root {
+  overflow: hidden;
+
+  .profile-root-description {
+    font-size: 40px;
+    text-align: center;
+    color:$charcoal-grey;
+    font-family: $font-family-2;
+    font-weight: $font-weight-light;
+  }
+
+  .profile-root-search-container {
+    padding: 90px 0;
+
+    @media only screen and (max-width: $breakpoint-foundation-small) {
+      padding: 60px 0;
+    }
+  }
+
+  .profile-root-actions {
+    margin-top: 70px;
+
+    @media only screen and (max-width: $breakpoint-foundation-small) {
+      margin-top: 50px;
+    }
+
+    text-align: center;
+  }
+
+  .profile-search {
+    display: inline-block;
+    vertical-align: bottom;
+    height: 45px;
+    width: 270px;
+
+    margin-left: 30px;
+    margin-top: 0;
+
+    @media only screen and (max-width: $breakpoint-foundation-small) {
+      margin-left: 0;
+      margin-top: 30px;
+    }
+
+    .search-input-field {
```

Comment:
Do we need to redefine the search-input-field here? I think the default variant is only used in this page, maybe we can add it there.